packages/filebrowser-extension/schema/browser.json

```
@@ -242,6 +242,18 @@
       "title": "Navigate files and directories with single click",
       "description": "Whether to allow single click selection on file browser items",
       "default": false
+    },
+    "autoOpenUploads": {
+      "type": "boolean",
+      "title": "Auto-open uploads",
+      "description": "Automatically open uploaded notebooks and files",
```

Let's make it explicit that non-supported file types will not be opened:

```suggestion
      "description": "Automatically open uploaded notebooks and supported files",
```